cpp/oneapi/dal/table/backend/homogen_table_impl.hpp

```diff
@@ -71,8 +71,8 @@ class homogen_table_impl
         return meta_;
     }
 
-    const void* get_data() const override {
-        return data_.get_data();
+    array<byte_t> get_data() const override {
```

Comment:
You already have `get_data_array()` on 86 line. Probably, it is needed to be removed or propagated to the base interface like `get_data()`.

Reply:
Removed